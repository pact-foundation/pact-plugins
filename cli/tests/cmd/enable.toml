bin.name = "pact-plugin"
args = "enable --help"
