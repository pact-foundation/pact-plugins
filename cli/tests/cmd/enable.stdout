Enable a plugin version

Usage: pact-plugin enable <NAME> [VERSION]

Arguments:
  <NAME>     Plugin name
  [VERSION]  Plugin version. Not required if there is only one plugin version

Options:
  -h, --help  Print help
